.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
you sure its working like this ?? comments are not supported in standard json file.
If its working, you can keep it like this.